drivers/mysql.go

```diff
@@ -352,20 +437,21 @@ func (db *MySQL) ExecuteQuery(query string) (results [][]string, err error) {
 	return
 }
 
-// TODO: Rewrites this logic to use the primary key instead of the id
-func (db *MySQL) UpdateRecord(table, column, value, primaryKeyColumnName, primaryKeyValue string) error {
-	table = db.formatTableName(table)
-	query := fmt.Sprintf("UPDATE %s SET %s = \"%s\" WHERE %s = \"%s\"", table, column, value, primaryKeyColumnName, primaryKeyValue)
-	_, err := db.Connection.Exec(query)
+func (db *MySQL) UpdateRecord(database, table, column, value, primaryKeyColumnName, primaryKeyValue string) error {
+	query := "UPDATE "
+	query += db.formatTableName(database, table)
+	query += fmt.Sprintf(" SET %s = ? WHERE %s = ?", column, primaryKeyColumnName)
+
+	_, err := db.Connection.Exec(query, value, primaryKeyValue)
 
 	return err
 }
 
-// TODO: Rewrites this logic to use the primary key instead of the id
-func (db *MySQL) DeleteRecord(table, primaryKeyColumnName, primaryKeyValue string) error {
-	table = db.formatTableName(table)
-	query := fmt.Sprintf("DELETE FROM %s WHERE %s = \"%s\"", table, primaryKeyColumnName, primaryKeyValue)
-	_, err := db.Connection.Exec(query)
+func (db *MySQL) DeleteRecord(database, table, primaryKeyColumnName, primaryKeyValue string) error {
+	query := "DELETE FROM "
+	query += db.formatTableName(database, table)
+	query += fmt.Sprintf(" WHERE %s = ?", primaryKeyColumnName)
+	_, err := db.Connection.Exec(query, primaryKeyValue)
```

Comment:
```suggestion
	query := fmt.Sprintf("DELETE FROM %s WHERE %s = ?", db.formatTableName(database, table), primaryKeyColumnName)
	_, err := db.Connection.Exec(query, primaryKeyValue)
```

Reply:
The thing is gosec G201 rule, you think it should be disabled?

Reply:
i think you fixed it the right way.

https://securego.io/docs/rules/g201-g202

I'm unsure if there is "clean way" when you are sprintf to set the table name

You may look at ORM how they do.

But now you understood how to write clean SQL queries with ? and arguments, maybe you could add a //nolint:gosec // disable it because of usage of table

Reply:
Thanks for suggestion. I did look at how query builders like [squirrel](https://github.com/Masterminds/squirrel) do it, and definitely they are not using Sprintf, they build the query writing to a "&bytes.Buffer{}" ([see](https://github.com/Masterminds/squirrel/blob/1ded5784535dcffa4e175d4efbd1ca2706927758/insert.go#L53-L115)). 

That's a good way, maybe i'll do it.

Reply:
Using a buffer is another way to contact string, it doesn't help at all with SQL injection.

You can use the buffer of course, as it's fast.

I think that's why they are using it

Reply:
By the time i'm more concerned about readability than SQL Injection, i think it does not makes sense (unless i'm missing something, if i do, please advice) to think that's a problem on a sql client, you will not self inject sql.

Reply:
You are correct, using parameters we avoid those things.

Thank you, sir. :)